metrics/cli/test/sentry/adapter/sentry_cli_service_adapter_test.dart

```diff
@@ -0,0 +1,594 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:cli/sentry/adapter/sentry_cli_service_adapter.dart';
+import 'package:cli/sentry/cli/sentry_cli.dart';
+import 'package:cli/sentry/constants/sentry_constants.dart';
+import 'package:cli/sentry/model/sentry_project.dart';
+import 'package:cli/sentry/strings/sentry_strings.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../../test_utils/file_helper_mock.dart';
+import '../../test_utils/matchers.dart';
+import '../../test_utils/prompter_mock.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group('SentryCliServiceAdapter', () {
+    const appPath = 'appPath';
+    const buildPath = 'buildPath';
+    const configPath = 'configPath';
+    const releaseName = 'releaseName';
+    const organizationSlug = 'organizationSlug';
+    const projectSlug = 'projectSlug';
+    const dsn = 'dsn';
+    const enterOrganizationSlug = SentryStrings.enterOrganizationSlug;
+    const enterReleaseName = SentryStrings.enterReleaseName;
+
+    final sentryCli = _SentryCliMock();
+    final prompter = PrompterMock();
+    final fileHelper = FileHelperMock();
+    final file = _FileMock();
+    final sentryService = SentryCliServiceAdapter(
+      sentryCli: sentryCli,
+      prompter: prompter,
+      fileHelper: fileHelper,
+    );
+    final sentryProject = SentryProject(
+      organizationSlug: organizationSlug,
+      projectSlug: projectSlug,
+    );
+    final stateError = StateError('test');
+    final environment = <String, dynamic>{
+      SentryConstants.dsn: dsn,
+      SentryConstants.releaseName: releaseName,
+    };
+
+    tearDown(() {
+      reset(sentryCli);
+      reset(prompter);
+      reset(fileHelper);
+      reset(file);
+    });
+
+    String enterProjectSlug() =>
+        SentryStrings.enterProjectSlug(organizationSlug);
+    String enterDsn() => SentryStrings.enterDsn(organizationSlug, projectSlug);
+
+    PostExpectation<String> whenAskOrganisationSlug() {
```

Comment:
```suggestion
    PostExpectation<String> whenPromptOrganisationSlug() {
```
What do you think? The same suggestion for all similar methods.